### PR TITLE
Add inventory dataset content

### PR DESCRIPTION
**Pull request by andrewemark**
Adds new inventory dataset content.

Note: I fixed a typo in the spills readme. Not sure where this needs to be propagated.

**Important:** At present, I am directly using the spills visualization code in the inventory notebook! This is not ideal, but neither is copy-pasting the spills visualization functions into a few file in infinity-tools. I assume we want to update infinity-tools to deduplicate, but I'm not sure what else relies on the visualization code in Infinity Tools. Let's discuss @aweitz. It needs to get figured out before releasing this dataset.

### CODE REVIEW

File: smartfacility-inventory-basic/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
++ A variety of camera positions with heights varying from ground-level to ceiling-level
+
+## Getting Started 
+Download the dataset from the [Infinity Marketplace](https://marketplace.infinity.ai/products/smart-facility-inventory-basic-dataset), and use the provided [quickstart notebook](quickstart.ipynb) to visualize labels.
+
+## Contents
+For each dataset sample, we provide the following files:
+
+- `{video_number}_params.json`: A JSON file describing the parameters used to generate the dataset sample. See [below](#dataset-parameters) for a description of each.
+- `{video_number}.mp4`: The RGB video of the spill.
```
Comment (aweitz) [inline smartfacility-inventory-basic/README.md:32]: Update for inventory dataset.

File: smartfacility-inventory-basic/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Dataset Parameters
+
+The `{video_number}_params.json` file provided for each dataset sample includes the parameters that were used to generate that specific video. For convenience, we provide a description of each parameter below:
+
+- `scene`: Environment/scene identity
+- `num_frames`: Number of frames in the video 
+- `image_width`: Image width in pixels
+- `image_height`: Image height in pixels
+- `camera_x`: Camera x position in world coordinates
```
Comment (aweitz) [inline smartfacility-inventory-basic/README.md:44]: Let's add units for these (camera_x/y/z).

File: smartfacility-inventory-basic/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+### Instance-level annotations
+Instance-level annotations are provided for the inventory items and all other objects in the scene. Instance-level annotations are accessible via the top-level `annotations` field of the JSON data structure. 
+
+**Note: The instance-level annotations enumerated below are only provided when a non-empty bounding box can be defined.** Annotations for items out of camera field of view are available, but they won't contain these keys.
+
+* `color`: Normalized RGB value in the corresponding instance segmentation masks.
+* `bbox`: Bounding box in standard COCO format.
+* `segmentation`: Polygon segmentation in standard COCO format.
+* `area`: Area enclosed by polygon segmentation.
```
Comment (aweitz) [inline smartfacility-inventory-basic/README.md:68]: The instance-level annotations also include `cuboid_coordinates`. You can copy the description from the robotics README.

File: smartfacility-inventory-basic/README.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+- `camera_y`: Camera y position in world coordinates
+- `camera_z`: Camera height in world coordinates
+- `random_seed`: The random seed used to generate the video
+
+Note that while the camera _location_ varies across videos, the focus location does not change.
+
+## Annotations
+
+The `{video_number}_labels.json` file is provided in standard [COCO](https://cocodataset.org/#format-data) format. The third-party [pycocotools](https://pypi.org/project/pycocotools/) package provides useful utility functions for working with the data structure.
+
```
Comment (aweitz) [inline smartfacility-inventory-basic/README.md:54]: The annotations for the inventory dataset also include video-level annotations (e.g. camera matrices) that were not included in the spills dataset. Can you add that section and update accordingly? You can copy formatting from the robotics README.

For consistency with the other dataset READMEs, the "Content" section should also be updated:  "A COCO-formatted JSON file of **video-**, frame-, and instance-level...”

**Review by aweitz — CHANGES_REQUESTED** [review]
Looks great! I added a few specific comments to the README that should be addressed before merging. Other miscellaneous comments:

- I think it would be nice to include both categorical and instance segmentation masks in the teaser image. I've attached an updated version here if you want to use it.
- Re: using the `spills` submodule, I would suggest we address this in a future PR (which likely involves a refactor to `infinity_tools`). The robotics quickstart notebook uses the `visionfit` submodule, so this would not be the only instance of re-using submodules. These datasets share defining features that make the visualization tools reusable. Namely, fitness and robotics are both human-centric, whereas spills and inventory are not. 
- A minor suggestion for the quickstart notebook: In the first demo, when we load metadata into a dataframe, I typically point to the entire dataset (rather than just the included sample data). This makes the demo a bit more interesting. I just define the path on my local machine, and then replace with a placeholder before committing.

inventory_teaser

**Comment by aweitz** [comment]
> Note: I fixed a typo in the spills readme. Not sure where this needs to be propagated.

I can address any outstanding typos in my PR.

File: smartfacility-inventory-basic/quickstart.ipynb [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "fecba5d8",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# Smart Facility: Inventory Dataset Basic - Demo Notebook\n",
```
Comment (aweitz) [inline smartfacility-inventory-basic/quickstart.ipynb:12]: I think this should technically be "Smart Facility: Inventory Basic Dataset"?

File: smartfacility-inventory-basic/README.md (hunk 3 above)
Comment (aweitz) [reply smartfacility-inventory-basic/README.md]: Can you add the schematic as shown [here](https://github.com/toinfinityai/infinity-datasets/blob/d6468e44142b64d023257f800127b3607c020681/smartfacility-robotics-basic/README.md?plain=1#L92-L102)?

**Review by aweitz — APPROVED** [review]

File: smartfacility-inventory-basic/README.md (hunk 3 above)
Comment (andrewemark) [reply smartfacility-inventory-basic/README.md]: Done

**Pull request merged**